services/core/functions/add-repo/index.ts

```diff
@@ -47,6 +49,19 @@ export const main = async (
       { conditions: { attr: "userId", exists: true } }
     );
 
+    // eslint-disable-next-line @typescript-eslint/no-misused-promises
+    inputBody.reposAdded.forEach( async (repo): Promise<void> => {
```

Comment:
are you sure that reposAdded is correct? Have you worked out how to test this?

Reply:
I'm not sure, just choose an appropriate name for the input variable. Is there a way of checking what the correct name would be?